Add "How to bridge tokens" pages (#300)
OffchainLabs/arbitrum-docs: arbitrum-docs/devs-how-tos/bridge-tokens/how-to-bridge-tokens-standard.mdx

  l1Signer: l1Wallet,
  l2Provider: l2Provider,
})
```

As mentioned before, you can also call the method `outboundTransferCustomRefund` manually in the router contract and specify the following parameters:

- address `_token`: L1 address of the ERC20 token being bridged
- address `_refundTo`: Account to be credited with the excess gas refund in L2
- address `_to`: Account to be credited with the tokens in L2
- uint256 `_amount`: Amount of tokens to bridge
- uint256 `_maxGas`: Max gas deducted from user’s L2 balance to cover the execution in L2
- uint256 `_gasPriceBid`: Gas price for the execution in L2
- bytes `_data`: 2 pieces of data encoded:
    - uint256 `maxSubmissionCost`: Max gas deducted from user's L2 balance to cover base submission fee
    - bytes `extraData`: “0x”

## Step 5: Wait for execution in L2

After calling the deposit method (or the outboundTransferCustomRefund, if you’re going the manual way), you’ll have to wait a bit until the message is executed in L2. We will verify the status of the underlying retryable ticket created to bridge the tokens. Check this page, to know more about [L1-to-L2 messages, also known as retryables](/arbos/l1-to-l2-messaging.mdx).

> You can programmatically wait for the execution of the transaction in L2 using Arbitrum’s SDK. You should first wait for the execution of the submission transaction (the one sent to the router contract) and then the execution of the L2 transaction.

Reviewer: ```suggestion
You can programmatically wait for the execution of the transaction on L2 using Arbitrum’s SDK. You should first wait for the execution of the submission transaction (the one sent to the router contract) and then the execution of the L2 transaction.
```

```tsx
/**
 * Now we wait for L1 and L2 side of transactions to be confirmed
 */
const depositRec = await depositTx.wait()
const l2Result = await depositRec.waitForL2(l2Provider)

/**
 * The `complete` boolean tells us if the l1 to l2 message was successful
 */
l2Result.complete
  ? console.log(
      `L2 message successful: status: ${L1ToL2MessageStatus[l2Result.status]}`
    )
  : console.log(
      `L2 message failed: status ${L1ToL2MessageStatus[l2Result.status]}`
    )
```

> If you’re going the manual way, you can verify if the message has been executed in L2 through the [Retryables Dashboard](https://retryable-dashboard.arbitrum.io/). Just paste the hash of transaction submitted to the router contract and the tool will tell you whether it’s been redeemed or not.

Reviewer: ```suggestion
If you’re going the manual way, you can verify if the message has been executed on L2 through the [Retryables Dashboard](https://retryable-dashboard.arbitrum.io/). Just paste the hash of transaction submitted to the router contract and the tool will tell you whether it’s been redeemed or not.
```

> ## Step 6: Check the new token contract created in L2

Reviewer: ```suggestion
## Step 6: Check the new token contract created on L2
```

> Finally, let’s find the token contract that has been created in L2. 

Reviewer: ```suggestion
Finally, let’s find the token contract that has been created on L2. 
```

> Using Arbitrum’s SDK, you can call method [getL2ERC20Address](/sdk/classes/assetBridger_erc20Bridger.Erc20Bridger#getl2erc20address) of the  [ERC20Bridger](/sdk/classes/assetBridger_erc20Bridger.Erc20Bridger) class, which will return the address of the token contract in L2 which corresponds to the L1 token contract send as parameter.

Reviewer: ```suggestion
Using Arbitrum’s SDK, you can call method [getL2ERC20Address](/sdk/classes/assetBridger_erc20Bridger.Erc20Bridger#getl2erc20address) of the [ERC20Bridger](/sdk/classes/assetBridger_erc20Bridger.Erc20Bridger) class, which will return the address of the token contract in L2 that corresponds to the L1 token contract sent as parameter.
```
